pareto/strategic_water_management/run_strategic_model.py

```diff
@@ -131,6 +133,7 @@
         "pipeline_capacity": PipelineCapacity.input,
         "node_capacity": True,
         "water_quality": WaterQuality.false,
+        "build_units": BuildUnits.user_units,
```

Comment:
I wonder, should the default value here be `scaled_units` instead of `user_units`? I understand that the benefit of using `user_units` is reduced runtime for the script, but using `scaled_units` is the current default behavior for `run_strategic_model.py` so I'm not sure that we should change that. Personally, I anticipate that my preference will generally be to use the `scaled_units` option, despite the additional runtime.

Reply:
Agreed! this is addressed in recent commits.